src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -57,7 +57,7 @@ StartEventResponse startAttachScannedDocs(CaseDetails theCase) {
 
     @Nonnull
     @SuppressWarnings("squid:S1135")
-    //^ For 'TODO' warning
+        //^ For 'TODO' warning
```

Comment:
oops

Reply:
yeah .. auto indent from idea ... so I just removed it ( failed checkstyle )